14-reduxy-actions-reducers/README.md

```diff
@@ -0,0 +1,116 @@
+# Step 14 - Reduxy Actions and Reducers
+
+Our goal in [Step 13](../13-action-reducers) was to separate the app state from the display logic by moving the business logic into a set of actions (api calls) and reducers (state changes). This, however, wasn't in true reduxy fashion. The goal of the next two steps is make that final jump and turn this application into a full redux app.
+
+In order to do this we will need to separate out our `action-reducers` into two separate files: `actions/index.js` and `reducers/index.js`. Our `actions` will respond to user interactions and communicate to our `reducers` what changes need to occur. The `reducers` will listen for dispatched `actions`, and respond when appropriate, modifying the state as appropriate (the same as before!). This 'broadcasting' and 'listening' is achieved by wrapping our `actions` in a `dispatch()` function.
+
+The goal of this step in particular is to pull out the `actions` and `reducers` into their own specific files and set up the interaction between the two.
+
+One last caveat is our app is very API call heavy, so we need support for asynchronous actions, something vanilla `Redux` does not support out of the box. Enter `redux-thunk`! Redux thunk lets us call `actions` as `actionCreators` like so:
+```js
```

Comment:
I'm not sure if this example code is going to make sense to someone who's never even seen Redux before. I would save the explanation/details of `redux-thunk` for further below where you do the implementation

Reply:
That's fair, I'll break up the implementation on the actions a little more so that I can introduce this idea in better context.